src/components/categories-list/CategoriesList.jsx

```diff
@@ -39,7 +39,8 @@ const CategoriesList = () => {
       ...prevCategories,
       ...categoriesWithColors
     ])
-  }, [skip, limit, colorPairs])
+    setQuantity(skip + fetchedCategories.length)
```

Comment:
Can we call right here `setUrlSearchParams({ quantity: skip + fetchedCategories.length })`? In order to avoid adding `quantity` state.

Reply:
Done.